app/controllers/home_controller.rb

```diff
@@ -64,7 +64,7 @@ def ordergroup
   # cancel personal memberships direct from the myProfile-page
   def cancel_membership
     if params[:membership_id]
-      membership = @current_user.memberships.find!(params[:membership_id])
+      membership = @current_user.memberships.find(params[:membership_id])
```

Comment:
i think a `find!` method does not exists? 
```
     NoMethodError:
       undefined method `find!' for #<ActiveRecord::Associations::CollectionProxy []>
       Did you mean?  find
```
find_by!` returns a `ActiveRecord::RecordNotFound` instead of `nil` compared to `find_by`...
I added another test to  check that `find` also returns a `ActiveRecord::RecordNotFound` Exception